Baikal/Kernels/CL/path_tracing_estimator.cl

```diff
@@ -242,11 +276,11 @@ KERNEL void FilterPathStream(
 
         if (Path_IsAlive(path))
         {
-            bool kill = (length(Path_GetThroughput(path)) < CRAZY_LOW_THROUGHPUT);
+            bool kill = (length(Path_GetThroughput(path)) < CRAZY_LOW_THROUGHPUT) || (isects[global_id].shapeid < 0);
```

Comment:
You changed kill-logic here. Please, describe why it was changed.

Reply:
If we don't kill them, we'll skip missed paths in GatherOpacity kernel, because there we consider killed paths.